### ISSUE TITLE
Grid size 

### ISSUE THREAD

**Issue by madratman** [cmusatyalab/mega-nerf#5]
Trying to reproduce the paper results. 
What should be the grid size argument for Mill19 (and other datasets)?

**Comment by hturki** [comment]
We used 8 GPUs for most of the experiments in the paper, with a grid size of "2 4" for all datasets except sci-art (for which we use "4 2"). I just added links to pretrained models and the cluster masks we used to the README. Let me know if you run into any issues!

**Comment by madratman** [comment]
Thanks! Yeah, I was running out of mem for anything more than "4 4" on a 16G V100, so 2 4 sounds good. 
Curious - were you use 8x 16 / 32G V100s for training?

**Comment by hturki** [comment]
yep!
